### ISSUE TITLE
Saving to disk should be optional

### ISSUE THREAD

**Issue by robbiewareham** [wkeeling/selenium-wire#316]
Would it be possible to only save the requests to memory rather than also saving to disk?

I would be interesting in submitting a PR but wanted to understand whether this is a feature that would be considered.

By reducing the I/O, so we can reduce the impact on performance, and as my use case in within a short lived Docker container, I am not sure what is gained by saving to disk in this case

**Comment by wkeeling** [comment]
Thanks for raising this.

A memory-only request storage implementation has been suggested before, so I'd certainly be open to it. It does make sense for improving performance in short-lived containers as you say.

I guess once implemented, a user could switch to the memory based storage with an option - e.g.:
```python
options = {
    'storage': 'memory',
}

driver.Chrome(seleniumwire_options=options)
```
with the current disk based storage being the default when the option isn't supplied.

Happy to receive a PR, but I should be able to look at it myself in the next few weeks.

**Comment by trovdimi** [comment]
This would be a really nice feature! In the meantime, a workaround on Linux would be to just mount the `request_storage_base_dir` in memory using `tmpfs`.

**Comment by wkeeling** [comment]
Now available in v4.4.0. See [the docs](https://github.com/wkeeling/selenium-wire#request-storage) for more info.
